response_operations_ui/templates/reporting-unit.html

```diff
@@ -4,7 +4,7 @@
 {% block main %}
 
 
-{% include 'partials/info-box.html' %}
+{% include 'partials/flashed-messages.html' %}
```

Comment:
Breaking acceptance tests by changing this. Need to add in a success id

Reply:
I've pushed acceptance test changes to match

Reply:
I had to delete my branch on AT to get it working, was really weird